### PR TITLE
fix tests for change in default RNG on nightly

### PR DESCRIPTION
**Pull request by palday**

### CODE REVIEW

File: test/sampling.jl [hunk 1]
```diff
@@ -219,11 +219,14 @@ wv = Weights([zeros(5); 1:4; -1])
 @test_throws ErrorException sample(a, wv, 1, replace=false)
 
 #### weighted sampling with dimension
-Random.seed!(1);
 
-@test sample([1, 2], Weights([1, 1]), (2,2)) == ones(2,2)
+# shape
+@test size(sample([1, 2], Weights([1, 1]), (2,2))) == (2, 2)
```
Comment (ararslan) [inline test/sampling.jl:224]: IMO testing the shape is not that useful, as the requested shape is literally part of the function call.

Comment (palday) [reply test/sampling.jl:224]: that's pretty much what the test did previously ... I think part of it is determine that it is indeed returning the requested shape

Comment (palday) [reply test/sampling.jl:224]: but I guess also that is tested in the other tests as well, so we could ditch it

Comment (ararslan) [reply test/sampling.jl:224]: Nothing better than deleting tests if they fail on nightly

**Review by ararslan — APPROVED** [review]

**Pull request merged**